packages/strapi-plugin-upload/admin/src/components/InputMedia/ErrorMessage.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+import { colors } from '@buffetjs/styles';
+
+const ErrorMessage = styled.p`
+  font-size: 1.3rem !important;
+  padding-top: 10px;
```

Comment:
In order to keep the baseline can you change the padding-top to `5px` in this component and add a `margin: 0;` property?

Also, could you update the `./Wrapper` component as follow so the baseline is kept in case of error:

```
import styled from 'styled-components';

const Wrapper = styled.div`
  margin-bottom: ${({ hasError }) => (hasError ? '1.7rem' : '2.3rem')};
`;

Wrapper.defaultProps = {
  hasError: false,
};

export default Wrapper;
```